src/main/java/com/hashedin/redmask/service/SnowflakeMaskingService.java

```diff
@@ -85,12 +91,14 @@ public void generateSqlQueryForMasking() {
        */
       for (int i = 0; i < config.getRules().size(); i++) {
         MaskingRule rule = config.getRules().get(i);
-        buildQueryAndAppend(rule, writer, config, url, connectionProps);
+        buildQueryAndAppend(rule, writer, config, url, connectionProps, maskingMode);
       }
 
       // TODO add required permission to grant access to different users
       // Grant access to the masked view data to user.
-      grantAccessToMaskedData(writer, config.getUser());
+      if (maskingMode.equals("dynamic")) {
```

Comment:
should not `dynamic` this be a constant?

Reply:
Add a comment about why you do grant access only incase of dynamic.

Reply:
Done